README.md

```diff
@@ -57,6 +57,35 @@ export METRICS_UTILITY_REPORT_RHN_LOGIN="test_login"
 export METRICS_UTILITY_REPORT_COMPANY_BUSINESS_LEADER="BUSINESS LEADER"
 export METRICS_UTILITY_REPORT_COMPANY_PROCUREMENT_LEADER="PROCUREMENT LEADER"
 
+# Gather and store the data in provided SHIP_PATH directory under ./report_data subdir
+metrics-utility gather_automation_controller_billing_data --ship --until=10m
+
+# Build report for previous month unless it already exists. Report will be created under ./reports dir under SHIP_PATH dir.
+metrics-utility build_report
+
+
+Example with local directory storage CCSPv2 type:
+```
+# Set needed ENV VARs for data gathering
+export METRICS_UTILITY_SHIP_TARGET=directory
+export METRICS_UTILITY_SHIP_PATH=/awx_devel/awx-dev/metrics-utility/shipped_data/billing
+
+# Set extra ENV VARs for report generation purposes
+export METRICS_UTILITY_REPORT_TYPE=CCSPv2
```

Comment:
following 5 lines are duplicates

Reply:
yeah, but it makes sense to document it per type, regardless of which env vars are reused for each type of report